add_submission_offsets.py

```diff
@@ -0,0 +1,89 @@
+from __future__ import print_function
```

Comment:
I renamed this file to make its name make a bit more sense, and reformatted a few things. Take a look at the whitespace changes for a better idea of the formatting that we use. an ide or other tool can help you do this too

Reply:
Thanks for the changes. For whitespace, I see the change to two lines between functions, and also the two lines before the first function definition after the cli setup. Are there any other key whitespace things that I should be aware of?